front/src/applications/stdcm/components/SimulationReportSheet.tsx

```diff
@@ -0,0 +1,272 @@
+import IBMPlexSansItalic from 'assets/simulationSheet/fonts/IBM-Plex-Sans/IBMPlexSans-Italic.ttf';
+import IBMPlexSansMedium from 'assets/simulationSheet/fonts/IBM-Plex-Sans/IBMPlexSans-Medium.ttf';
+import IBMPlexSansRegular from 'assets/simulationSheet/fonts/IBM-Plex-Sans/IBMPlexSans-Regular.ttf';
+import IBMPlexSansSemiBold from 'assets/simulationSheet/fonts/IBM-Plex-Sans/IBMPlexSans-SemiBold.ttf';
+import iconAlert from 'assets/simulationSheet/icon_alert_fill.png';
+import logoSNCF from 'assets/simulationSheet/logo_sncf_reseau.png';
+import type { PostStdcmApiResponse, RollingStockWithLiveries } from 'common/api/osrdEditoastApi';
+
+import styles from './SimulationReportStyleSheet';
+import { formatDate, formatDay, getStopDurationTime, getStopTime } from '../utils';
+
+interface SimulationReportSheetProps {
+  stdcmData: PostStdcmApiResponse | null;
+  rollingStockData: RollingStockWithLiveries | undefined;
+  number: string;
+  mapCanvas?: string;
+}
+
+const SimulationReportSheet: React.FC<SimulationReportSheetProps> = ({
+  stdcmData,
+  rollingStockData,
+  number,
+  mapCanvas,
+}) => {
+  const { t } = useTranslation('stdcm-simulation-sheet');
+
+  Font.register({
+    family: 'IBM Plex Sans',
+    fonts: [
+      { src: IBMPlexSansRegular, fontWeight: 'normal' },
+      { src: IBMPlexSansMedium, fontWeight: 'medium' },
+      { src: IBMPlexSansSemiBold, fontWeight: 'semibold' },
+      { src: IBMPlexSansBold, fontWeight: 'bold' },
+      { src: IBMPlexSansItalic, fontStyle: 'italic' },
+    ],
+  });
+
+  Font.register({
+    family: 'IBM Plex Mono',
+    fonts: [
+      { src: IBMPlexMonoRegular, fontWeight: 'normal' },
+      { src: IBMPlexMonoMedium, fontWeight: 'medium' },
+      { src: IBMPlexMonoBold, fontWeight: 'bold' },
+    ],
+  });
+
+  return (
+    <Document>
+      <Page wrap={false} style={styles.page} size={[1344]}>
+        <View style={styles.alertBanner}>
+          <Image src={iconAlert} style={styles.alertIcon} />
+          <Text style={styles.simulationTitle}>{t('simulation')}</Text>
+          <Text style={styles.message}>{t('warningMessage')}</Text>
+        </View>
+        <View>
+          <View style={styles.numberDateBanner}>
+            <View style={styles.stdcm}>
+              <Text style={styles.title}>{t('stdcm')}</Text>
+              <Text style={styles.creation}>{t('stdcmCreation')}</Text>
+            </View>
+            <View style={styles.numericInfo}>
+              <Text style={styles.number}>
+                {t('number')}
+                {number}
+              </Text>
+              <Text style={styles.creationDate}>{formatDate(stdcmData?.path.created)}</Text>
+            </View>
+            <Image src={logoSNCF} style={styles.sncfLogo} />
+          </View>
+        </View>
+        <View style={styles.rcInfo}>
+          <View style={styles.rcBox}>
+            <Text style={styles.rcName}>Super Fret</Text>
```

Comment:
It would be better to instantiate a const fakeInformation to group all these information: rcName, rcPersonName, rcPhoneNumber, rcMail...

Reply:
good idea ! done